### ISSUE TITLE
[FE] 10-10CG - File Upload: 508-defect-1 ❗ Launchblocker: Representative documentation appears under supporting documentation, unable to edit.

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#24550]
# [508-defect-1 :exclamation: Launchblocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

Issue Title: 508 Defect 1, Launchblocker! Representative documentation appears under supporting documentation, unable to edit.

Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a user, I expect to be able to upload all necessary documents. I expect that once uploaded, documents appear in their assigned place and that I can edit them when necessary.

<h2>Details</h2>
When uploading documentation, I am presented with only the option to upload representative documentation. Once I do, the document is presented under "supporting documentation" and I am unable to delete it. I am also unable to edit representative documentation so that the form appears in the right place.

**NOTE: This issue may already be resolved.  Please check the steps and confirm**

<h2>Acceptance Criteria</h2>
 - [ ] The user is able to upload both representative and supporting documentation and they appear in their proper places on review.
 - [ ] Alternatively, the option to upload supporting documentation is removed completely.

Steps to Recreate
<ul>
<li> Access https://staging.va.gov/family-member-benefits/apply-for-caregiver-assistance-form-10-10cg/introduction</li>
<li> Choose "Begin your application."</li>
<li> complete steps 1-3 of the application</li>
<li>upload a PDF which is less than 25MB per the submission guidelines.</li>
<li>Proceed to step 5.</li>
<li>Attempt to edit the representative documentation.</li>
</ul>

**Comment by hdjustice** [comment]
I believe this issue may have been resolved with the Sign as a Representative improvements implemented after May 2021.   We will have the team review & test to confirm.

cc: @markfallows

**Comment by hdjustice** [comment]
Issue was resolved with the feature launched in Q2 2022
